src/compas_ags/utilities/displaysettings.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+
+__all__ = [
+    'calculate_drawing_scale',
+    'find_anchor_point',
+    'calculate_force_scale',
+]
+
+
+def calculate_drawing_scale(form, force):
```

Comment:
this is a detail but in general i would combine words that belong together and only use underscores to separate logical parts of the name. so here that would be `calculate_drawingscale`